ui/src/components/common/SlidingSidebar/index.tsx

```diff
@@ -43,9 +54,13 @@ export function SlidingSidebar({
   vertexDetailsProps,
   edgeDetailsProps,
   pipelineSpecsProps,
+  generatorDetailsProps,
+  errorsProps,
   onClose,
 }: SlidingSidebarProps) {
-  const [width, setWidth] = useState<number>(pageWidth ? pageWidth / 2 : 0);
+  const [width, setWidth] = useState<number>(
+    errorsProps ? 350 : pageWidth ? pageWidth / 2 : 0
```

Comment:
Use MIN_WIDTH_BY_TYPE map value here instead of hardcoded 350

Reply:
done